### DISCUSSION TITLE
Each gantt Task as a time picker

### DISCUSSION THREAD

**Discussion by n-arakawa** [marcusolsson/grafana-gantt-panel#81] [Ideas]
This is feature request.

I want to **abuse**  each 'gantt Task' as the time picker.
Add  the extra data-link which set the start and end time of the  'gantt Task' to dashboard time-range!

**Comment by marcusolsson** [comment]
I like it! Though I think you should already be able to do this using data links. Have you tried creating a link to the dashboard where you set the from and to query parameters?

**Comment by n-arakawa** [comment]
Data link URL is absolute(As far as I know), Therefore
- It is not portable(dashboard export and import)
- Invoke full reload

Hope to use locationsrv.

https://grafana.com/docs/grafana/latest/packages_api/runtime/locationsrv/
```
  getLocationSrv().update({
    query: {
      from: From_value,
      to: To_lue,
    },
    partial: true,
    replace: true
  })

```

**Comment by n-arakawa** [comment]
Another option:

https://grafana.com/docs/grafana/latest/packages_api/data/panelprops/

```
onChangeTimeRange({from:1000, to:2000})
```
